draft-ietf-httpbis-digest-headers.md

```diff
@@ -117,7 +117,7 @@ integrity protection; for instance TCP checksums or TLS records [RFC2818].
 
 However, there are cases where relying on this alone is insufficient. An
 HTTP-level integrity mechanism that operates independent of transfer can be used
-to detect programming errors and/or corruption of data at rest, be used across
+to detect programming errors and/or corruption of data in flight or at rest, be used across
```

Comment:
adding "in flight" seems werid. What errors do you expect that TLS integrity wouldn't have caught?

Reply:
It refers to trasfer-coding-like issues without mentioning them. See https://github.com/httpwg/http-extensions/issues/1208#issuecomment-634351643

Reply:
Yeah its fine to keep then